src/libraries/System.Text.Json/tests/Common/JsonCreationHandlingTests.Generic.cs

```diff
@@ -42,8 +42,15 @@ internal struct StructList<T> : IList<T>, IList
             set => _list[index] = (T)value;
         }
 
+        [JsonConstructor]
```

Comment:
Are we affecting test coverage in anyway by adding this attribute and the 2nd constructor? Basically, are we losing coverage for the case when the struct just has an unannotated parameterless ctor?

Reply:
It shouldn't, the particular suite tests the populate semantics of the containing type and does attempt to deserialize the collection itself. I added the annotation out of abundance of caution, so that the newly added constructor doesn't impact the serialization semantics of the collection type itself.